src/main/java/freiman/citibike/aws/CitibikeRequestHandler.java

```diff
@@ -17,16 +17,20 @@ public CitibikeResponse handleRequest(APIGatewayProxyRequestEvent event, Context
         String body = event.getBody();
         Gson gson = new Gson();
         CitibikeRequest request = gson.fromJson(body, CitibikeRequest.class);
-
         StationServiceFactory factory = new StationServiceFactory();
         StationService service = factory.getService();
-        Map<String, Station> stations = factory.merge(service);
-        StationFinder stationFinder = new StationFinder(stations);
+
+        StationCache cache = new StationCache();
```

Comment:
The goal of the cache is to reuse the data every time `handleRequest()` is called.

By creating this INSIDE handleRequest() you aren't actually caching anything.